### DISCUSSION TITLE
client.execute(command)

### DISCUSSION THREAD

**Discussion by muhammadbekzod** [dylankenneally/react-native-ssh-sftp#19] [Q&A]
I guess there some bug with using ros2 commander. please check it. 

code:     
let command =
      'ros2 topic pub --once /cmd_vel_key geometry_msgs/msg/Twist "{linear: { x: 0.3,  y: 0.0,  z: 0.0}, angular: {x: 0.0, y: 0.0, z: 0.0}}"';
    log(`about to execute '${command}'`);

**Comment by dylankenneally** [comment]
Sorry for the delay in responding @muhammadbekzod, I didn't see this discussion item until today.

You're comment doesn't really give me enough information to understand what you're challenge is - could you elaborate? Is it specific to running `ros2` commands, or are you having issues running standard commands (`ls` for example)?
